### ISSUE TITLE
Packaging (conda): CommandLine Error: Option 'disable-symbolication' registered more than once!

### ISSUE THREAD

**Issue by SylvainCorlay** [root-project/cling#440]
As I am working on getting cling 0.9 packaged for conda-forge.

I managed to successfully build cling, however, I am seeing the following failure at run time:

```
CommandLine Error: Option 'disable-symbolication' registered more than once!
```

**Comment by Axel-Naumann** [comment]
That's two sets of LLVM libraries getting symbol-confused. For ROOT we hide all of LLVM's symbols for that exact reason. What libraries are loaded into your cling, what's the OS etc?

**Comment by Axel-Naumann** [comment]
Is this still an issue, @SylvainCorlay ?

**Comment by SylvainCorlay** [comment]
Yes, I have not been able to put cycles into this recently unfortuntely.

**Comment by Axel-Naumann** [comment]
OK will leave it open then until you come back to us (or until it gets self-healed with a subsequent llvm update)

**Comment by SylvainCorlay** [comment]
@serge-sans-paille was able to figure it out, and patched the conda recipe for cling 0.9 in the following way:

https://github.com/conda-forge/cling-feedstock/pull/54/commits/f42b723d69bb8f88429c85fdfdff1a55460e92b9

**Comment by SylvainCorlay** [comment]
@Axel-Naumann let me know if this solution is useful to you.
